.gitignore

```diff
@@ -1,17 +1,25 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
+src/main/client/.yarn/cache
```

Comment:
We can potentially just use `src/main/client/.yarn*` to do a broader match instead of writing all of these lines?

Reply:
That's what I wanted to do but all the sample gitignore files and yarn gitignore docs I found ignored specific subfolders rather than the whole .yarn folder. It didn't help that we now have this yarn v1/v2 split, some would-be links to v1 documentation actually redirect to v2 docs, etc., so I'm having trouble specifically finding whether yarn v1's .yarn folder would ever still hold anything worth committing.

The gitignore file as committed does happen to result in my entire .yarn folder getting ignored. So, purely experimentally, this suggests that it should be safe to ignore the whole folder.

Reply:
I think it's fine to ignore the whole folder. It doesn't exist for me locally.